### PR TITLE
Fix a bug who made fire with brightness crash the plugin

### PR DESCRIPTION
**Pull request by AdriLez**

### CODE REVIEW

**Comment by mnkyking** [comment]
Does the MZ zip also need to be updated?

**Comment by AdriLez** [comment]
Why not, it has been a while and I found a stupid maker-related error in it some weeks ago.

**Comment by NerdyChara** [comment]
I take responsibility for that one. 😓

**Comment by AdriLez** [comment]
It is actually still bugged, there is a crash in case the hex number has only a single digit.

**Comment by AdriLez** [comment]
Fixed

File: Community_Lighting_MZ.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ var Community = Community || {};
...
 Community.Lighting.name = "Community_Lighting_MZ";
 Community.Lighting.parameters = PluginManager.parameters(Community.Lighting.name);
-Community.Lighting.version = 4.2;
+Community.Lighting.version = 4.3;
 var Imported = Imported || {};
 Imported[Community.Lighting.name] = true;
 /*:
 * @target MZ
-* @plugindesc v4.2 Creates an extra layer that darkens a map and adds lightsources! Released under the MIT license!
+* @plugindesc v4.3 Creates an extra layer that darkens a map and adds lightsources! Released under the MIT license!
```
Comment (mnkyking) [inline Community_Lighting_MZ.js:16]: We should have a version history markdown doc. Maybe I'll make one later.

Comment (NerdyChara) [reply Community_Lighting_MZ.js:16]: Version 4.3 is where my map notes were added. And the softer lights option I don't know about the rest though.

**Pull request merged**